templates/dsl/rest-api/restserviceimpl.java

```diff
@@ -3,8 +3,13 @@
 {{ai-type 'service'}}
 package {{packageName @root.options.basePackage}}.service;
 
-import java.util.*;
+import com.kapeta.spring.exceptions.IllegalArgumentException;
+import com.kapeta.spring.exceptions.InvalidStateException;
+import com.kapeta.spring.exceptions.NotFoundException;
```

Comment:
we could consider a * import, in-case it invents something new :)

Reply:
Sounds like a good idea, but this is to provide context.